canonicalwebteam/flask_base/log_utils.py

```diff
@@ -0,0 +1,143 @@
+import logging
+import datetime
+import json
+from functools import lru_cache
+
+from flask import Flask
+from rich.logging import RichHandler
+from rich.text import Text
+from pythonjsonlogger.json import JsonFormatter
+from pythonjsonlogger.core import RESERVED_ATTRS
+from gunicorn.glogging import Logger as GunicornLogger
+
+from canonicalwebteam.flask_base.env import get_flask_env
+from canonicalwebteam.flask_base.observability import get_trace_id
+
+
+DEFAULT_DEV_FORMAT = "[%(name)s] %(message)s"
+
+
+def _date_format_with_ms(dt: datetime.datetime) -> Text:
+    return Text(dt.strftime("%Y-%m-%d %H:%M:%S.%f")[:-3])
+
+
+class RequestFilter(logging.Filter):
```

Comment:
Can this class be renamed to include traceid in the name as it has a very specific role of injecting a trace id into a log rather than what the current name suggests: filtering requests.